components/indexer.py

```diff
@@ -1,30 +1,35 @@
 from magicbot import feedback
+import wpilib
 
 
 class Indexer:
     indexer_motors: list
     indexer_switches: list
+    ready_piston: wpilib.DigitalInput
 
     def on_enable(self) -> None:
+        for motor in self.indexer_motors:
+            motor.setInverted(True)
```

Comment:
Why is this in `on_enable`?

Reply:
Why are you asking passive-aggressive questions instead of making suggestions to improve the code?